pkg/os/exec.go

```diff
@@ -11,6 +11,7 @@ func RunWithPrivilege(cmdAndArgs ...string) (string, string, error) {
 	if err != nil {
 		return "", "", err
 	}
+	/* #nosec */
```

Comment:
Did you audit all RunWithPrivilege callers for that G204 issue?

Reply:
@cfergeau yes,  nothing is exposed to user arg side.

```bash
$ git grep RunWithPrivilege
pkg/crc/preflight/preflight_checks_darwin.go:   stdOut, stdErr, err := crcos.RunWithPrivilege("hdiutil", "attach", tempFilePath)
pkg/crc/preflight/preflight_checks_darwin.go:   stdOut, stdErr, err = crcos.RunWithPrivilege("installer", "-package", virtualBoxPkgLocation, "-target", "/")
pkg/crc/preflight/preflight_checks_darwin.go:   stdOut, stdErr, err = crcos.RunWithPrivilege("hdiutil", "detach", virtualBoxMountLocation)
pkg/crc/preflight/preflight_checks_darwin.go:   stdOut, stdErr, err := crcos.RunWithPrivilege("chown", "root:wheel", path)
pkg/crc/preflight/preflight_checks_darwin.go:   stdOut, stdErr, err = crcos.RunWithPrivilege("chmod", "u+s", path)
pkg/crc/preflight/preflight_checks_darwin.go:           stdOut, stdErr, err := crcos.RunWithPrivilege("mkdir", resolverDir)
pkg/crc/preflight/preflight_checks_darwin.go:   stdOut, stdErr, err := crcos.RunWithPrivilege("touch", resolverFile)
pkg/crc/preflight/preflight_checks_darwin.go:   stdOut, stdErr, err := crcos.RunWithPrivilege("chown", currentUser.Username, filename)
pkg/crc/preflight/preflight_checks_linux.go:    stdOut, stdErr, err := crcos.RunWithPrivilege("yum", "install", "-y", "libvirt", "libvirt-daemon-kvm", "qemu-kvm")
pkg/crc/preflight/preflight_checks_linux.go:    stdOut, stdErr, err := crcos.RunWithPrivilege(path, "enable", "libvirtd")
pkg/crc/preflight/preflight_checks_linux.go:    stdOut, stdErr, err := crcos.RunWithPrivilege("usermod", "-a", "-G", "libvirt", currentUser.Username)
pkg/crc/preflight/preflight_checks_linux.go:    stdOut, stdErr, err := crcos.RunWithPrivilege(path, "start", "libvirtd")
pkg/crc/preflight/preflight_checks_linux.go:    _, _, err := crcos.RunWithPrivilege("rm", "-f", oldLibvirtDriverPath)
pkg/crc/systemd/host.go:        stdOut, stdErr, err := crcos.RunWithPrivilege("systemctl", "daemon-reload")
pkg/crc/systemd/host.go:        stdOut, stdErr, err := crcos.RunWithPrivilege("systemctl", action.String(), name)
```